RecoPPS/RPixEfficiencyTools/README.md

```diff
@@ -1,3 +1,47 @@
+## Onboarding
+### Onboarding - how to setup workspace 
+1. login to ssh 
+```
+ssh <username>@lxplus.cern.ch
+# example
+ssh mobrzut@lxpllus.cern.ch
+```
+
+2. Go to your afs directory *(note1: this project was tested with afs not eos - probalby some problems with eos)*
+*(note2: check if you have afs enabled and at least 5Gb of storage [cern resources portal](https://resources.web.cern.ch/resources/Manage/ListServices.aspx))*
+
+```
+#example
+cd /afs/cern.ch/user/m/mobrzut
+```
+3. It's best to create a separate folder (ex. `mkdir automation`) and setup there a CMSSW. (follow `project setup` section on readme)
+
+### Onboarding - how to browse root files
+There are a bunch of methods to browse histograms of root file. 
+**Method 1**
+1. Connect to ssh with flag -XY
+```
+ssh -XY <username>@lxplus.cern.ch
+```
+
+2. Go to the directory with your file and open there root. 
+```
+root
+``` 
+3. Open TBrowser. Now GUI should appear. 
+```
+new TBrowser
+```
+
+4. You can exit from root by calling exit method 

```

Comment:
The standard command for exiting a root session is '.q', exit(0) works, but it's a bit more cumbersome and I'm not sure it allows root to exit gracefully as it should